HACKING.md

```diff
@@ -24,7 +24,9 @@ To build this project you must first install several command line utilities and
 - [`make`](https://www.gnu.org/software/make/) - Make build system
 - [`mvn`](https://maven.apache.org/index.html) (version 3.5 and above) - Maven CLI
 - [`helm`](https://helm.sh/) - Helm Package Management System for Kubernetes
-    - After installing Helm be sure to run `helm init`.
+    - Both Helm 2 and Helm 3 are supported. In order to use either two, it is recommended that you download both versions. A convenient way to do so is to download the [get_helm.sh](https://helm.sh/docs/intro/install/#from-script) script and specify the version you want to download either for Helm 2 or Helm 3.
```

Comment:
It's not just _recommended_ that both versions are installed, `make helm_install` or `make all` will _fail_ if you don't have a `helm3` on your path. And since `helm3` is not the normal name of the binary, meeting these environmental requirements is actually quite a high barrier for new contributors to get over. I wonder if we can make the build:

* Pass (maybe with a warning) if there is no `helm`
* Cope with the non-presence of `helm2` and `helm3`
* If there is a `helm` detect its version and use that if it's 2 or 3

Reply:
I will give that a go and come back.

Reply:
When discussing with Jakub, the recommendation was to have as the default Helm version to be 3. Hence, the modification of `helm_install: helm-charts/helm3` in the main `Makefile`. The renaming of the binaries to `helm2` and `helm3` are done only for Travis. So, may I assume here that the wording needs changing only and mandate to have Helm 3 installed? In that case, I can put a check for the version only - as per the third bullet point - and verify that the version is not < 2.*.